src/fetch.js

```diff
@@ -51,14 +51,28 @@ exports.fetchData = async (configOptions, reporter, cache, contentTypeOption) =>
 };
 
 
-exports.fetchContentTypes = async (config, contentTypeOption) => {
+exports.fetchContentTypes = async (config, contentTypeOption, cache) => {
   try {
     config.cdn = config.cdn ? config.cdn : 'https://cdn.contentstack.io/v3';
+    const typePrefix = config.type_prefix || 'Contentstack';
+  
+    const lastFetchedTimeCacheKey = `${typePrefix.toLowerCase()}-content-type-fetch-time-${config.api_key}`;
```

Comment:
Instead of making your own key each time, what about a function that returns a key?
```js
const getCacheKey = (name, config) => `${config.type_prefix ?? 'Contentstack'}-${name}-${config.api_key}`;
```

The benefit here is when you want to update the structure (for example i dont think api key is enough in a cache key, due to #117.) it's updated in a single place.